<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>425</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>MV Boli</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Nova Light</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>General Commands:-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Exit">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Options:-</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>541</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="2">
      <widget class="QPushButton" name="RemoveFile">
       <property name="font">
        <font>
         <family>Rockwell</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Remove the File</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="Date">
       <property name="font">
        <font>
         <family>Rockwell</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="SoftwareMangement">
       <property name="font">
        <font>
         <family>Rockwell</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Software Mangement</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="CurrentLocation">
       <property name="font">
        <font>
         <family>Rockwell</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>See Current Location</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="StopService">
       <property name="font">
        <font>
         <family>Rockwell</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Stop A Service</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="StartService">
       <property name="font">
        <font>
         <family>Rockwell</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Start A Service</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="CopyFile">
       <property name="font">
        <font>
         <family>Rockwell</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Copy File to Another Directory</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="MoveFile">
       <property name="font">
        <font>
         <family>Rockwell</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Move File to Another Directory</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="Calender">
       <property name="font">
        <font>
         <family>Rockwell</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Calender</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="AdvanceCommands">
       <property name="font">
        <font>
         <family>Rockwell</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Advance Commands</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="rad">
       <property name="font">
        <font>
         <family>Rockwell</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Remove A Directory</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="kcu">
       <property name="font">
        <font>
         <family>Rockwell</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Know Current User</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
